.travis.yml

```diff
@@ -14,6 +14,7 @@ script:
 after_success:
   - bash <(curl -s https://codecov.io/bash)
   - semantic-release
+  - yarn deploy-storybook --out=demo
```

Comment:
This is not working. See https://travis-ci.org/github/conveyal/transitive.js/builds/744310503#L338.